tests/test.bats

```diff
@@ -32,13 +79,15 @@ teardown () {
 
 @test "Default database present" {
 	[[ $SKIP == 1 ]] && skip
+	_healthcheck_wait
```

Comment:
`_healthcheck_wait` is only necessary after `make start`. Please remove it here.

Reply:
fixed